.github/workflows/reviewdog.yml

```diff
@@ -0,0 +1,15 @@
+name: reviewdog
+on: [pull_request_target]
+
+jobs:
+  golangci-lint:
+    name: runner / golangci-lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code into the Go module directory
```

Comment:
the name should be updated

Reply:
deleted.

**the check result title like this**: reviewdog / golangci (pull_request_target) Failing after 1s — reviewdog [golangci] report